backend/remote/backend_common.go

```diff
@@ -316,6 +316,10 @@ func (b *Remote) costEstimate(stopCtx, cancelCtx context.Context, op *backend.Op
 				b.CLI.Output(b.Colorize().Color("Waiting for cost estimate to complete..." + elapsed + "\n"))
 			}
 			continue
+		case "skipped_due_to_targeting": // TEMP: not available in the go-tfe library yet; will update this to be tfe.CostEstimateSkippedDueToTargeting once that's available.
```

Comment:
We're planning to submit this `tfe.CostEstimateSkippedDueToTargeting` upstream to `go-tfe` asynchronously from getting this PR in. Will update this to use the named constant in a subsequent PR once it's included in a `go-tfe` release.